app/build.gradle

```diff
@@ -63,6 +63,10 @@ android {
   lintOptions {
     checkReleaseBuilds false
   }
+
+  dexOptions {
+    javaMaxHeapSize "4g"
```

Comment:
I'm pretty sure this option is no longer used and may simply be ignored. I'm having trouble finding where I saw that though.

Reply:
Does this fix some build issue that you were seeing?